### ISSUE TITLE
When running ng test in an ng-cli project, it fails to show sourcemaps for the debug window

### ISSUE THREAD

**Issue by williamBurgeson** [angular/angular-cli#6892]
### Bug Report or Feature Request (mark with an `x`)
```
- [ x] bug report -> please search issues before submitting
- [ ] feature request
```

### Versions.
Occurs from 1.1.2 onwards (1.0.1 and 1.1.1 seem OK, didn't test intervening versions)

### Repro steps.
Simply create a new ng cli project (ng new my-project), go into package.json and change the version of @angular/cli to 1.1.2. Note I am using yarn. To ensure "isolation", I have deleted node_modules and yarn.lock and then run yarn install each time. 
I run ng test on the command line, then click the debug button in the browser and the open the dev tools (chrome on windows). In sources, I expect to see webpack:// with the source files but from 1.1.2 this item is absent.

### The log given by the failure.
n/a

### Desired functionality.
I want to see the sourcemapped original ts files in the browser dev tools (Chrome on Windows). Obviously if I am building an angular app using TDD it makes it very difficult when I can't debug into my application code when I need to do so.

### Mention any other details that might be useful.
I noticed this on a project I am working on, but created a new ng cli blank project to compare to, which worked fine out of the box. I isolated the problem down to version 1.1.2 onwards. 
Obviously this might be down to issues with webpack or other dependencies, however from my point of view as a "user" the broken version is 1.1.2+, and until this is resolved I am unable to upgrade past 1.1.1

**Comment by devoto13** [comment]
Looks like duplicate of #6737, #6785 and #6640. Should be fixed by #6862.

**Comment by Brocco** [comment]
Closing as a duplicate. Thanks @devoto13 for finding the other related issues.
